test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -2275,30 +2455,41 @@ class MultipleDisposalsTriageControllerSpec
       "show a form error" when {
 
         "the user submits nothing" in {
-          val answers = IncompleteMultipleDisposalsTriageAnswers.empty.copy(
-            individualUserType = Some(Self),
-            numberOfProperties = Some(2),
-            wasAUKResident     = Some(false),
-            countryOfResidence = Some(country),
-            assetTypes         = None
-          )
-          val (session, _) = sessionDataWithStartingNewDraftReturn(answers)
+          List(trustDisplay, agentDisplay, normalDisplay).foreach { displayType: TrustAgentOrNormalDisplay =>
+            withClue(
+              s"For user type ${displayType.userType} with ${displayType.name.fold[String](_ => "Trust Name", _ => "Individual Name")} "
+            ) {
+              val answers = IncompleteMultipleDisposalsTriageAnswers.empty.copy(
+                individualUserType = Some(Self),
+                numberOfProperties = Some(2),
+                wasAUKResident     = Some(false),
+                countryOfResidence = Some(country),
+                assetTypes         = None
+              )
+              val session = sessionDataWithStartingNewDraftReturn(
+                answers,
+                displayType.name,
+                displayType.userType == UserType.Agent
```

Comment:
don't use `==` - it's not typesafe. Use the cats `===` instead

Reply:
Done.

Reply:
I still see `==` here - if you do a search for `== ` you'll see all the instances where it needs updating

Reply:
I did a replace all but probably hadnt worked or sth. Done now. sorry.